db/migrate/20230221171922_create_users.rb

```diff
@@ -3,8 +3,8 @@ def change
     create_table :users do |t|
       t.text :first_name, null: false
       t.text :last_name, null: false
-      t.text :email, null: false, index: { unique: true }
-      t.text :password, null: false
+      #t.string :email, null: false, index: { unique: true }
```

Comment:
We shouldn't be changing previous migration files, we should be creating new ones to modify the schemas.